ch12/handle_segv.c

```diff
@@ -197,9 +200,17 @@ int main(int argc, char **argv)
 		exit(1);
 	}
 
+	/*Setting up the alternate stack to properly handle the SIGSEGV*/
```

Comment:
Nice!
just fix code style as per Linux kernel style guidelines
Tip- use 'indent -linux <fname.c>' to do so ..